vpr/src/route/route_timing.cpp

```diff
@@ -2052,8 +2053,9 @@ static void evaluate_timing_driven_node_costs(t_heap* to,
     }
 
     //Update the backward cost (upstream already included)
-    to->backward_path_cost += (1. - cost_params.criticality) * cong_cost; //Congestion cost
-    to->backward_path_cost += cost_params.criticality * Tdel;             //Delay cost
+    to->backward_path_cost += (1. - cost_params.criticality) * cong_cost;       //Congestion cost
+    to->backward_path_cost += cost_params.criticality * Tdel;                   //Delay cost
+    to->backward_path_cost += cost_params.criticality * switch_penalty_cost;    //Penalty cost
```

Comment:
I would add the penalty without `criticality *`.

Reply:
Done